### ISSUE TITLE
examples/add_points_on_nD_shapes.py appears broken — camera should be set to an oblique angle

### ISSUE THREAD

**Issue by jni** [napari/napari#4160]
## 🐛 Bug

I'm not clear on what exactly it's meant to show — @kevinyamauchi do you have an idea of what it's demonstrating? — but at any rate the resulting image is weird at best:

<img width="1610" alt="Screen Shot 2022-02-28 at 7 10 37 pm" src="https://user-images.githubusercontent.com/492549/155947381-23bcf29f-5fe4-4c2a-a1d3-d1f88b3dc216.png">

It also generates the following warning:

```pytb
/Users/jni/conda/envs/all/lib/python3.9/site-packages/numpy/core/numeric.py:2453: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  return bool(asarray(a1 == a2).all())
```

**Comment by jni** [comment]
... Never mind! It's working great, I just needed to rotate the camera — meaning probably the example should rotate the camera too? And we should fix our 3D shapes rendering so that the shape edges don't look so weird.

[Link to code](https://github.com/napari/napari/blob/842008f6df909bcbc391789badd7f2414eed6ec4/examples/add_points_on_nD_shapes.py)

<img width="1610" alt="Screen Shot 2022-02-28 at 7 13 34 pm" src="https://user-images.githubusercontent.com/492549/155947761-35804a7e-393e-4839-bfe4-cf4e77853b88.png">

**Comment by jni** [comment]
btw what made it click was that the history showed *both* @kevinyamauchi and @alisterburt had messed with it 😂 y'all are awesome! 😂

**Comment by alisterburt** [comment]
I'm super glad this *isn't* borked - debugging these things is never fun 😆 PR incoming

**Comment by jni** [comment]
Closed by #4161
